asset/css/doc.css

```diff
@@ -47,19 +47,24 @@ div.odoc .spec a.anchor::after {
   padding: 0.6em;
 }
 
-div.odoc *:hover > a.anchor {
+/* div.odoc *:hover > a.anchor {
   opacity: 1;
 }
 
 div.odoc *:hover > a.anchor:hover {
   color: rgb(75 85 99);
+} */
+
+div.odoc *>a.anchor {
+  color: rgb(75 85 99);
```

Comment:
seems unrelated to the package navigation documentation navigation

Reply:
Initially, I thought it was related to the hover CSS of navigation line, so I made changes to it. However, upon checking again, I realized that it wasn't that css, as you rightly pointed out. It is a mistake which I will revert with lib indicator css issue.